benchmarks/dense_matmul/genbenchmark.py

```diff
@@ -84,17 +93,20 @@ def output_log(output_report) -> str:
     result = "# Dense Matmul Benchmark Results\n\n"
     dt_string = datetime.now().strftime("%d/%m/%Y %H:%M:%S")
     result += f"This test was run at {dt_string}\n\n"
-    for layout in ("cyclic", "banked"):
-        result += f"Results for a {layout} layout \n\n"
-        result += "| benchmark | layout | M | N | K | plots | cycles | ideal | utilization |\n"
-        result += "| --- | --- | --- | --- | --- | --- | --- | --- | --- |n"
+    for layout, add_c in itertools.product(("cyclic", "banked"), (True, False)):
+        result += f"Results for a {layout} layout {'with add C' if add_c else ''} \n\n"
+        result += "| benchmark | layout | add C | M | N | K | plots | cycles | ideal | utilization |\n"
+        result += "| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |\n"
```

Comment:
I have just rediscovered pandas dataframes and I really regret not using them sooner, together with seaborn it allows you to do some very fast and beautiful (interactive I think even) plotting. Should also make it a lot easier to create tables such as the ones here. Not for this PR

Reply:
Yes, that is indeed probably a much better solution, they ven seem to have `.to_markdown` functions for nice printing

Reply:
seaborn is probably not allowed on github pages, idk